src/constants/routes.js

```diff
@@ -0,0 +1,7 @@
+export const routes = {
+  home: '/',
```

Comment:
all object fields should be in SCREAMING_SNAKE_CASE

Reply:
Done.